exercises/leap/.meta/hints.md

```diff
@@ -0,0 +1,4 @@
+## Hints
```

Comment:
I was thinking that perhaps this should be something else in this case, maybe "Notes"  or something like that?

Reply:
Makes sense to me... fixed!